src/cljs/proton/layers/base.cljs

```diff
@@ -10,3 +12,43 @@
 (defmulti get-keybindings dispatch)
 (defmulti get-keymaps dispatch)
 (defmulti describe-mode dispatch)
+
+(defn register-package-dependencies [key deps]
```

Comment:
What's the distinction between `register-package-dependencies` and `register-layer-dependencies`.

It seems like we are only using the latter. What is the intended purpose for `register-package-dependencies`?